Clarify how to configure Ruler to send alerts to Alertmanager (#3685)
grafana/mimir: docs/sources/mimir/operators-guide/architecture/components/alertmanager.md

When an Alertmanager starts, it attempts to load the alerts state for a given tenant from other Alertmanager replicas. If the load from other Alertmanager replicas fails, the Alertmanager falls back to the state that is periodically stored in the storage backend.

In the event of a cluster outage, this fallback mechanism recovers the backup of the previous state. Because backups are taken periodically, this fallback mechanism does not guarantee that the lastest state is restored.

## Ruler configuration

> [Ruler]({{< relref "ruler/index.md" >}}) must be configured with the addresses of Alertmanagers with the `-ruler.alertmanager-url` flag.

Reviewer: ```suggestion
You must configure the [ruler]({{< relref "ruler/index.md" >}}) with the addresses of Alertmanagers via the `-ruler.alertmanager-url` flag.
```

Reviewer: Rationale: Use active voice where possible. Also, I changed "with" to "via" here to read more smoothly; this change is a word choice change.

> The address should point to Alertmanager's API.

Reviewer: ```suggestion
Point the address to Alertmanager’s API.
```
The term "should" is generally ambiguous; don’t be afraid to write directly to the reader.

> Alertmanager's API prefix is configured by the `-http.alertmanager-http-prefix` flag, and defaults to `/alertmanager`.

Reviewer: ```suggestion
You can configure the Alertmanager’s API prefix via the `-http.alertmanager-http-prefix` flag, which defaults to `/alertmanager`.
```

> For example, if Alertmanager is listening at `https://mimir-alertmanager.namespace.svc.cluster.local` and is using the default API prefix, set `-ruler.alertmanager-url` to `https://mimir-alertmanager.namespace.svc.cluster.local/alertmanager`.

Reviewer: ```suggestion
For example, if Alertmanager is listening at `https://mimir-alertmanager.namespace.svc.cluster.local`, and it is using the default API prefix, set `-ruler.alertmanager-url` to `https://mimir-alertmanager.namespace.svc.cluster.local/alertmanager`.
```
